lazycatlabs/flutter_auth_app
# fix: resolve deprecated code

## What it does

- update the library version
- regenerate the freezed class
- update documentation for recommended version

## How to test

1. clone the project
2. run `flutter analyze --no-fatal-infos --no-fatal-warnings`
3. observe the result

## Screenshot

![image](https://github.com/lazycatlabs/flutter_auth_app/assets/1531684/dc9c1da2-286b-4b1a-b667-00120ee91c73)
